bin/gwdetchar-scattering

```diff
@@ -259,6 +271,18 @@ page.add('A summary of triggers that occur during active scattering '
          'segments is recorded in CSV format here:')
 page.a(os.path.basename(summfile), href=summfile, target='_blank')
 page.p.close()
+# link X_TR_B file.
```

Comment:
@siddharth101, in general, single-clause comment lines should not end with full stops:

```suggestion
# link X_TR_B file
```